### PR TITLE
Ana/5 filter transactions on page transactions

### LINKED ISSUE #5: Filter Transactions on PageTransactions
It should be possible to filter transactions on the PageTransactions according to type. Best use case is filtering only market-related transactions, so users can easily access their order history

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #5

Filter transactions according to type

image

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Now with a proper placeholder:

image

**Comment by mdyring** [comment]
Looks good, comments:

1) Can we align input field with width of table (similar to validators page)?
2) Having "Unknown" as an option seems a bit messy, can that option be removed?
3) Optional for bonus points, supporting a date range as well would be sweet. Maybe in same line as the dropdown?

**Comment by mdyring** [comment]
4) Please add "Your Transactions" to top of page to make it consistent with the other pages. (also nicer UX I think)

**Comment by Bitcoinera** [comment]
> Optional for bonus points, supporting a date range as well would be sweet. Maybe in same line as the dropdown?

I think we can convert this one into another issue / PR, otherwise it can get messy

**Comment by Bitcoinera** [comment]
all suggestions except the bonus points have been applied

File: pages/transactions/index.vue [hunk 1]
```diff
@@ -78,7 +78,7 @@ const lunieMessageTypes = [
   },
   {
     value: `ClaimRewardsTx`,
-    key: `ClaimRewards`,
+    key: `Claim Rewards`,
```
Comment (Bitcoinera) [inline pages/transactions/index.vue:81]: Yes, better like that :+1:

**Pull request merged**